config/monitoring/prometheus/service-account.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: metrics-reader
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  annotations:
+    kubernetes.io/service-account.name: metrics-reader
+  name: metrics-reader
+type: kubernetes.io/service-account-token
```

Comment:
Isn't this secret specific for the configuration in [infra-deployments](https://github.com/redhat-appstudio/infra-deployments)? If so, then such changes should go there.

Reply:
What's the difference between [the service monitor defined in infra-deployments](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build-service/base/monitoring.yaml#L26-L43) and the service monitor defined in this repository?  I see both getting deployed to clusters controlled by infra-deployments, but they both seem to be targeting the same endpoints.

Would it be safe to remove one of them, i.e. migrate the usage of the one from infra-deployments into this repository?  As it stands, a deployment that tries to use this repository as a source for deployment manifests is going to have a broken service monitor.

Reply:
in the cluster in build-service namespace I see 2 service monitors:

build-service:
based on https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build-service/base/monitoring.yaml#L27

build-service-controller-manager-metrics-monitor:
based on https://github.com/konflux-ci/build-service/blob/main/config/monitoring/prometheus/monitor.yaml